test/behavior/enum.zig

```diff
@@ -104,13 +104,6 @@ test "single field non-exhaustive enum" {
 }
 
 test "enum type" {
-    const foo1 = Foo{ .One = 13 };
-    const foo2 = Foo{
-        .Two = Point{
-            .x = 1234,
-            .y = 5678,
-        },
-    };
```

Comment:
I don't think this behavior test should be deleted, just put some simple asserts in there that the values are intact.